yarn.lock

```diff
@@ -0,0 +1,9805 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
We are using NPM for the backend and I think we should keep that consistent in all repos if there is no reason for which Yarn is needed